salt/states/user.py

```diff
@@ -557,28 +556,6 @@ def present(
                 name,
             )
 
-    # Warn until Silicon release, when old gid_from_name argument is used.
-    # Since gid_from_name is the only thing that we're pulling from the kwargs,
-    # we can also remove **kwargs from the function definition once we remove
-    # the entire if block below. The following two tests will also become
-    # redundant when this block is cleaned up:
-    #
-    # integration.states.test_user.UserTest.test_user_present_gid_from_name
-    # integration.states.test_user.UserTest.test_user_present_gid_from_name_and_usergroup
-    gid_from_name = kwargs.pop("gid_from_name", None)
```

Comment:
we need a `*.removed` changelog since we are removing `gid_from_name` in place of `usergroup`

Reply:
Ok

Reply:
Done